app/src/main/java/no/usn/mob3000/ui/components/info/ContentEditor.kt

```diff
@@ -0,0 +1,85 @@
+package no.usn.mob3000.ui.components.info
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import no.usn.mob3000.R
+import no.usn.mob3000.Viewport
+
+/**
+ * Content editor for news, documentation and faq.
+ *
+ * @author frigvid
+ * @contributor 258030

```

Comment:
Note, author should not be changed when simply extracting existing code with minimal changes.